imports/ui/pages/player/player.html

```diff
@@ -1,5 +1,5 @@
 <template name="player">
-  <div id="player-container" class="player-container {{ hideControls }} {{ descriptionClass }} {{ mutedClass }} {{ volumeClass }} {{ playPause }}">
+  <div id="player-container" class="player-container {{ hideControls }} {{ descriptionClass }} {{ mutedClass }} {{ volumeClass }} {{ playPause }}" tabIndex=99>
```

Comment:
This allows the container itself to be focusable and in turn receive keyboard-related events